### PR TITLE
docs: Add troubleshooting guideline for local setup - PIP Version and Missing Python Headers

### PR DESCRIPTION
**Pull request by klub-jayanth**
I have updated the contribution docs with two resolution of errors for local setup.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -112,6 +112,35 @@ Steps to set up a development environment:
     ```
 - Note: this does not start any of the celery schedulers or workers, which are needed if you want to run any analytics.
 
+#### *Troubleshooting*
+a> If you come across cases where python libraries are failing with this error
```
Comment (Samyak2) [inline CONTRIBUTING.md:116]: Can you use markdown lists here instead? Like:
```suggestion
1. If you come across cases where python libraries are failing with this error 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -112,6 +112,35 @@ Steps to set up a development environment:
...
 
+#### *Troubleshooting*
+a> If you come across cases where python libraries are failing with this error 
+ ```
+ERROR: Failed building wheel for numpy
+ Failed to build pyarrow numpy
+ERROR: Could not build wheels for pyarrow, numpy which use PEP 517 and cannot be installed directly
+```
+
+Resolution -> Upgrade your pip version
```
Comment (Samyak2) [inline CONTRIBUTING.md:123]: Adding the command for upgrading pip here will be useful.

Comment (Samyak2) [inline CONTRIBUTING.md:123]: To keep the formatting consistent with the rest of the document:
```suggestion
Resolution: Upgrade your pip version 
```

**Review by Samyak2 — CHANGES_REQUESTED** [review]
Thank you for the contribution!

Looks good overall just a few minor formatting issues.

**Comment by klub-jayanth** [comment]
Will be sending the PR with the updated changes from personal account.

**Pull request closed without merging**